modules/RTC/TraceablePeerConnection.js

```diff
@@ -422,6 +422,8 @@ export default function TraceablePeerConnection(
         }, 1000);
     }
 
+    this._lastVideoSenderUpdatePromise = null;
```

Comment:
You could initialize this to Promise.resolve(), then you can just chain the promises allways.